ansible/playbook/tasks/roles/clickhouse/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: Remove old clickhouse packages
+  yum:
+    state: absent
+    name:
+      - percona-clickhouse-client
+      - percona-clickhouse-server
+      - percona-clickhouse-common-static
+      - percona-clickhouse-server-common
```

Comment:
please ensure that old data won't be deleted during upgrade.

Reply:
@nikita-b this removal should not delete data directories, right?

Reply:
@BupycHuk @atymchuk It's only remove binaries and packages metadata. I tested the update and it works fine.